cats/cats-core/src/main/java/com/netflix/spinnaker/cats/cache/CompositeCache.java

```diff
@@ -70,6 +70,21 @@ public CacheData get(String type, String id, CacheFilter cacheFilter) {
         return allItems.values();
     }
 
+    @Override
+    public Collection<String> existingIdentifiers(String type, Collection<String> ids) {
+        HashSet<String> identifiers = new HashSet<>(ids.size());
+        HashSet<String> remainingIds = new HashSet<>(ids);
+        for (Cache cache : caches) {
+          Collection<String> existing = cache.existingIdentifiers(type, remainingIds);
+          identifiers.addAll(existing);
+          remainingIds.removeAll(existing);
```

Comment:
maybe a comment in here that we're basically short-circuiting once we've found a match for an `id` in one of the caches?